utils.py

```diff
@@ -166,24 +156,7 @@ def post_crash(url, payload, headers, size, compressed=False):
     )
 
 
-def generate(metadata=None, dumps=None):
-    """Returns raw_crash, dumps"""
-    raw_crash = {
-        'ProductName': 'Firefox',
-        'Version': '1',
-    }
-    if dumps is None:
-        dumps = {
-            'upload_file_minidump': b'abcd1234'
-        }
-
-    if metadata is not None:
-        raw_crash.update(metadata)
-
-    return raw_crash, dumps
```

Comment:
We don't send in additional metadata, so I folded this into `generate_sized_crashes`.